{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":20,"msg":"Sync server version: 1.3.8 2022-02-25 10:58:11 -0600 50659ecda06a8b79a41f24c1fdff05187324402b","time":"2024-11-05T18:38:02.915Z","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":30,"msg":"Acquiring actors for session 1730831883: ACTOR1","time":"2024-11-05T18:38:02.921Z","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":30,"msg":"Session 1730831883 with label \"oneApp-login 2\" started at 2024-11-05 18:38:02 UTC","time":"2024-11-05T18:38:02.927Z","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":30,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.6774937Z","msg":"Actor 29586 of type ACTOR1 was acquired by test session 1730831883","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":20,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.6903073Z","msg":"DEBUG: Test actor JAR versions:","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":20,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.6923014Z","msg":"DEBUG:   opentest-appium-0.0.1.jar: 2022-02-25 10:58 -0600 0.0.1-50659ecda06a8b79a41f24c1fdff05187324402b-dirty","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":20,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.6933026Z","msg":"DEBUG:   opentest-base-0.0.1.jar: 2022-02-25 10:58 -0600 0.0.1-50659ecda06a8b79a41f24c1fdff05187324402b-dirty","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":20,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.6962999Z","msg":"DEBUG:   opentest-selenium-0.0.1.jar: 2022-02-25 10:58 -0600 0.0.1-50659ecda06a8b79a41f24c1fdff05187324402b-dirty","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":40,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.7003057Z","msg":"WARN: Failed to log extension JARs","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":10,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.7013039Z","msg":"TRACE: Firing SESSION_STARTED event...","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":30,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.8848741Z","msg":"Including script file \"common-scripts\\csvTools.js\"...","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":50,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:03.9135003Z","msg":"ERROR: java.lang.RuntimeException: Failed to evaluate the \"dataSet\" property of the data-driven test as a JavaScript expression. The expression we evaluated was >>> (function() {\n  $include(\"common-scripts/csvTools.js\")\n  var urlData = $readYaml(\"data/config.yaml\")\n  var loginData = $csvToJson(\"data/py-data/login-mail/login-mail.csv\")\n  return {\n    url: urlData,\n    data: loginData\n  }\n})()\n <<<.\r\n\tat org.getopentest.base.TestActor.evalDataSet(TestActor.java:1274)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:2975)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3113)\r\n\tat org.getopentest.Main.main(Main.java:20)\r\nCaused by: java.lang.RuntimeException: Failed to run script file common-scripts\\csvTools.js\r\n\tat org.getopentest.base.TestActor.includeScriptFile(TestActor.java:2601)\r\n\tat org.getopentest.base.TestActor.lambda$evalIncludes$6(TestActor.java:1343)\r\n\tat java.base/java.util.Arrays$ArrayList.forEach(Arrays.java:4390)\r\n\tat org.getopentest.base.TestActor.evalIncludes(TestActor.java:1342)\r\n\tat org.getopentest.base.TestActor.access$600(TestActor.java:84)\r\n\tat org.getopentest.base.TestActor$15.accept(TestActor.java:629)\r\n\tat jdk.scripting.nashorn.scripts/jdk.nashorn.internal.scripts.Script$Recompilation$95$1$\\^eval\\_/0x000000010039b840.L:1(<eval>:2)\r\n\tat jdk.scripting.nashorn.scripts/jdk.nashorn.internal.scripts.Script$Recompilation$94$\\^eval\\_/0x0000000100397040.:program(<eval>:1)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:655)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:513)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:527)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:456)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:413)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:409)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:162)\r\n\tat java.scripting/javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)\r\n\tat org.getopentest.base.TestActor.evalDataSet(TestActor.java:1272)\r\n\t... 3 more\r\nCaused by: javax.script.ScriptException: <eval>:6:52 Expected , but found =>\r\n    return csv().fromString(csvData).then((jsonObj) => {\r\n                                                    ^ in <eval> at line number 6 at column number 52\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:477)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.compileImpl(NashornScriptEngine.java:544)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.compileImpl(NashornScriptEngine.java:531)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.evalImpl(NashornScriptEngine.java:409)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.eval(NashornScriptEngine.java:162)\r\n\tat java.scripting/javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)\r\n\tat org.getopentest.base.TestActor.includeScriptFile(TestActor.java:2598)\r\n\t... 19 more\r\nCaused by: jdk.nashorn.internal.runtime.ParserException: <eval>:6:52 Expected , but found =>\r\n    return csv().fromString(csvData).then((jsonObj) => {\r\n                                                    ^\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.AbstractParser.error(AbstractParser.java:297)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.AbstractParser.error(AbstractParser.java:282)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.AbstractParser.expectDontAdvance(AbstractParser.java:353)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.AbstractParser.expect(AbstractParser.java:340)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.argumentList(Parser.java:3716)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.leftHandSideExpression(Parser.java:3430)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.unaryExpression(Parser.java:4440)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.expression(Parser.java:4620)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.conditionalExpression(Parser.java:4772)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.assignmentExpression(Parser.java:4711)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.expression(Parser.java:4589)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.expression(Parser.java:4585)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.returnStatement(Parser.java:2338)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.statement(Parser.java:1066)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.sourceElements(Parser.java:909)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.functionBody(Parser.java:4206)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.functionExpression(Parser.java:3831)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.statement(Parser.java:1099)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.sourceElements(Parser.java:909)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.program(Parser.java:844)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.parse(Parser.java:325)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.parser.Parser.parse(Parser.java:285)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.Context.compile(Context.java:1500)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.Context.compileScript(Context.java:1467)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.internal.runtime.Context.compileScript(Context.java:750)\r\n\tat jdk.scripting.nashorn/jdk.nashorn.api.scripting.NashornScriptEngine.compileImpl(NashornScriptEngine.java:542)\r\n\t... 24 more\r\n","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":30,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:04.0592225Z","msg":"Executing segment 1 of test logueo-mail/logueo-mail...","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":30,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:04.0614414Z","msg":"Evaluating action 1/4 (org.getopentest.selenium.NavigateTo) in segment 1 (Navegar a oneApp)...","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":50,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:04.108037Z","msg":"ERROR: java.lang.Exception: Failed executing test logueo-mail/logueo-mail, segment 1\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2208)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3077)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3113)\r\n\tat org.getopentest.Main.main(Main.java:20)\r\nCaused by: java.lang.RuntimeException: There was an error while executing action no. 1 (org.getopentest.selenium.NavigateTo) in segment 1 of test logueo-mail/logueo-mail.\nThe action definition was:\n{\n  \"action\": \"org.getopentest.selenium.NavigateTo\",\n  \"args\": {\n    \"url\": \"$dataRecord.url.LoginOneAppPROD.url\"\n  },\n  \"description\": \"Navegar a oneApp\"\n}\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2142)\r\n\t... 3 more\r\nCaused by: java.lang.NullPointerException\r\n\tat org.getopentest.base.TestActor.executeActionByDef(TestActor.java:1839)\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2092)\r\n\t... 3 more\r\n","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":30,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:04.6485107Z","msg":"Test session has completed","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":10,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:04.6573018Z","msg":"TRACE: Firing TEST_COMPLETED event...","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":10,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:04.6579847Z","msg":"TRACE: Firing SESSION_COMPLETED event...","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":30,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:38:04.6599883Z","msg":"Actor 29586 of type ACTOR1 with no tags is waiting to be acquired by a test session...","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":10,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:39:58.8626041Z","msg":"TRACE: Initializing test session information in the \"announce\" method...","v":0}
{"name":"1730831883","hostname":"DESKTOP-8V10OGQ","pid":32624,"level":10,"actorType":"ACTOR1","actorId":"29586","time":"2024-11-05T18:39:58.8636041Z","msg":"TRACE: Session ID in sync server response was 1730831998; actor session ID was null","v":0}
